pkg/aws/ocpgwdeployer.go

```diff
@@ -142,6 +149,21 @@ func (d *ocpGatewayDeployer) Deploy(input api.GatewayDeployInput, status reporte
 	return nil
 }
 
+func (d *ocpGatewayDeployer) setSubnetSuffixes(publicSubnets []types.Subnet) {
+	pattern := fmt.Sprintf(`%s.*-subnet-public-%s.*`, regexp.QuoteMeta(d.aws.infraID), regexp.QuoteMeta(d.aws.region))
+	re := regexp.MustCompile(pattern)
+
+	for i := range publicSubnets {
+		tags := publicSubnets[i].Tags
+		for i := range tags {
+			if strings.Contains(*tags[i].Key, "Name") && re.MatchString(*tags[i].Value) {
+				d.aws.nodeSGSuffix = "-node"
```

Comment:
Maybe it would be useful for debugging to log the calculated SG suffixes

Reply:
Done.

Reply:
Done.

Reply:
Had to remove it since the logic is now moved to initialization. But the names will be printed in other logs.